.github/workflows/on-pull-request-opened-and-synchronize.yml

```diff
@@ -77,35 +79,149 @@ jobs:
         if: ${{ failure() }}
         with:
           name: coverage
-          path: dist/coverage/
+          path: dist/coverage
+
+  fetch-visual-snapshots:
+    name: Fetch Visual Snapshots
+    runs-on: ubuntu-latest
+    steps:
+      # TODO: Remove after this is resolved: https://www.cloudflarestatus.com/incidents/t5nrjmpxc1cj
+      - name: Install AWS CLI
```

Comment:
I wanted to abstract all of this into a composite action. But it turned out to be about the same because I would have had to pass the secrets to the action.